src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsUrlTest.java

```diff
@@ -190,7 +190,7 @@ public void characters(char[] ch, int start, int length) {
 
         @Override
         public void endElement(String uri, String localName, String qName) {
-            if (singleCheckNameList != null && qName.equals(NODE_NAME)) {
```

Comment:
this a bug to check and pmd (if rule active), right?

If you want to care lets open issue.